components/PokemonListRow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import { colors } from '../style/styleVariables';
+
+interface Props {
+  pokemonId: string;
+  pokemonName: string;
+  onPokemonListRowClicked: (pokemonName: string) => void;
+}
+
+function propsAreEqual(prevProps: Props, nextProps: Props) {
+  return (
+    prevProps.pokemonId === nextProps.pokemonId &&
+    prevProps.pokemonName === nextProps.pokemonName
+  );
+}
+
+export function PokemonListRow({
+  pokemonId,
+  pokemonName,
+  onPokemonListRowClicked
+}: Props) {
+  const onPress = () => {
```

Comment:
Theoretically it could be wrapped into `useCallback`, but since you use `React.memo` then `useCallback` would be recreated anyway when `React.memo` forces rerender as it uses almost all props, so there's no need